businessRules/donations.js

```diff
@@ -0,0 +1,49 @@
+const { v4 } = require('uuid');
+const { createDonation } = require('../models/donations');
+const { updateIdea, getAllIdeas } = require('../models/ideas');
+
+function donationData(firstName, sum, userId, ideaId) {
```

Comment:
It's not clear what this function does. It doesn't return anything and the name doesn't communicate its purpose. Try to refactor it to improve readability.

Reply:
Renamed, refactored and tested.